charts/ingress-nginx/tests/controller-service_test.yaml

```
@@ -0,0 +1,32 @@
+suite: test controller service
+templates:
+  - controller-service.yaml
+
+tests:
+  - it: should not create any service by default
+    set:
+      controller.service.enabled: false
```

```suggestion
      controller.service.external.enabled: false
```
Mostly a preparation. This flag got introduced some time ago and hopefully we might be able to move all the external Service related values from `controller.service` to `controller.service.external` one day.